src/main/java/com/flansmod/common/guns/GunType.java

```diff
@@ -243,7 +247,8 @@
 	/**
 	 * For guns with 3D models
 	 */
-	@SideOnly(Side.CLIENT)
+	//TODO properly separate the data
```

Comment:
Has this been tested on dedicated servers? Turning on some of this stuff can cause runtime classNotFoundExceptions

Reply:
since i removed it, i don't think that there should be any problem concerning this. But i tested all of this on a server so there is definitely no problem